packages/server/postgres/queries/getMattermostAuthByUserIdTeamId.ts

```diff
@@ -0,0 +1,18 @@
+import getPg from '../getPg'
+import {
+  getMattermostAuthByUserIdTeamIdQuery,
+  IGetMattermostAuthByUserIdTeamIdQueryResult
+} from './generated/getMattermostAuthByUserIdTeamIdQuery'
+
+// this table has a composite primary key (userId, teamId),
+// which cannot use the index with a WHERE IN or JOIN on VALUES
+// so if we want to query multiple userIds/teamIds, just call this multiple times
```

Comment:
fun fact, i tested this again, it's not true! postgres can use an index on a composite primary key if there are enough records for it to do so. my first test only had a few. i tested again with ~1000 records & it used the index. The more you know 🌠

Reply:
I'll strike the comment in both places!

Reply:
(granted, the better thing to do would be to refactor this to fetch multiple...but I am going to leave this alone this will change in a subsequent change set)